tests/run.sh

```diff
@@ -204,6 +204,12 @@ for f in $tests; do
     # Send the test file to awesome.
     cat $f | DISPLAY=$D "$AWESOME_CLIENT" 2>&1
 
+    # Kill awesome after 1 minute (e.g. with errors during test setup).
+    (sleep 60; if kill -0 $awesome_pid 2>/dev/null; then
+        echo "Killing (stale?!) awesome (PID $awesome_pid) after 10 seconds."
+        kill $awesome_pid
+      fi) &
+
```

Comment:
Seems like a good idea, but on the other hand that might fail already somehow.

I think the `kill` is safe, since it's really supposed to be the PID of awesome.